std_msgs/Header header
int64           l_error_code
int64           l_hardware_status_code
string          x_description
